### PR TITLE
Revert "Revert "Added employee stuff-user story one""

### PR DESCRIPTION
**Pull request by AdamB-Kainos**
Reverts AdamB-Kainos/java-dropwizard-flyway-starter#16

### CODE REVIEW

File: src/main/java/org/example/controllers/EmployeeController.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public Response createEmployee(final EmployeeRequest employeeRequest) {
+        try {
+            return Response
+                    .status(Response.Status.CREATED)
+                    .entity(employeeService.createEmployee(employeeRequest))
+                    .build();
+        } catch (FailedToCreateException | SQLException e) {
+            return Response.serverError().build();
+        } catch (InvalidException e) {
+            throw new RuntimeException(e);
```
Comment (shaunganley) [inline src/main/java/org/example/controllers/EmployeeController.java:37]: this should return a 400 bad request

File: src/main/java/org/example/controllers/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    EmployeeService employeeService;
+
+    public EmployeeController(final EmployeeService employeeService) {
+        this.employeeService = employeeService;
+    }
+
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response createEmployee(final EmployeeRequest employeeRequest) {
```
Comment (shaunganley) [inline src/main/java/org/example/controllers/EmployeeController.java:28]: requirement 1 asks for the ability to create a delivery employee but this appears to be a generic employee?

File: src/main/java/org/example/validators/EmployeeValidator.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import org.example.exceptions.Entity;
+import org.example.exceptions.InvalidException;
+import org.example.models.EmployeeRequest;
+
+public class EmployeeValidator {
+
+    public void validateEmployee(final EmployeeRequest employeeRequest)
+            throws InvalidException {
+
+        if (employeeRequest.getSalary() < 1) {
```
Comment (shaunganley) [inline src/main/java/org/example/validators/EmployeeValidator.java:12]: any other check we should be doing here?

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/daos/DatabaseConnector.java [hunk 4]
```diff
@@ -6,8 +6,12 @@
 
 public final class DatabaseConnector {
     private static Connection conn;
-    private DatabaseConnector() { }
-    public static Connection getConnection() throws SQLException {
+
+    private DatabaseConnector() {
+    }
+    @SuppressWarnings("checkstyle:LineLength")
```
Comment (boydstepkainos) [inline src/main/java/org/example/daos/DatabaseConnector.java:12]: How come we're suppressing line length checks?

**Pull request closed without merging**